src/Microsoft.DocAsCode.Build.Engine/ManifestUtility.cs

```diff
@@ -26,8 +27,12 @@ public static void RemoveDuplicateOutputFiles(ManifestItemCollection manifestIte
                               .GroupBy(obj => obj.relativePath, FilePathComparer.OSPlatformSensitiveStringComparer)
                               .Where(g => g.Count() > 1))
             {
-                Logger.LogWarning($"Overwrite occurs while input files \"{string.Join(", ", duplicates.Select(duplicate => duplicate.item.SourceRelativePath))}\" writing to the same output file \"{duplicates.Key}\"");
-                itemsToRemove.UnionWith(duplicates.Skip(1).Select(duplicate => duplicate.item.SourceRelativePath));
+                var duplicateSources = duplicates.Skip(1).Select(duplicate => duplicate.item.SourceRelativePath).ToList();
+                // TODO: plan to change this warning to error, add error code to analyze the impact.
+                Logger.LogWarning(
+                    $"Multiple input files are attempting to write to the same output file \"{duplicates.Key}\". Input file \"{duplicates.First().relativePath}\" is selected as the output content, please rename other input files to avoid duplicate output files: \"{string.Join(", ", duplicateSources)}\".",
```

Comment:
Yes you're right, I'll update the msg.

---
In reply to: [112098532](https://github.com/dotnet/docfx/pull/1561#discussion_r112098532) [](ancestors = 112098532)